homeassistant/components/isy994/const.py

```diff
@@ -417,6 +421,29 @@
     "118": UnitOfPressure.HPA,
     "119": UnitOfEnergy.WATT_HOUR,
     "120": UnitOfVolumetricFlux.INCHES_PER_DAY,
+    "122": CONCENTRATION_MICROGRAMS_PER_CUBIC_METER,  # Microgram per cubic meter
+    "123": f"bq/{UnitOfVolume.CUBIC_METERS}",  # Becquerel per cubic meter
+    "124": f"pCi/{UnitOfVolume.LITERS}",  # Picocuries per liter
+    "125": "pH",
+    "126": "bpm",  # Beats per Minute
+    "127": UnitOfPressure.MMHG,
+    "128": "J",
+    "129": "BMI",  # Body Mass Index
+    "130": f"{UnitOfVolume.LITERS}/{UnitOfTime.HOURS}",
+    "131": SIGNAL_STRENGTH_DECIBELS_MILLIWATT,
+    "132": "bpm",  # Breaths per minute
+    "133": UnitOfFrequency.KILOHERTZ,
+    "134": f"{UnitOfLength.METERS}/{UnitOfTime.SECONDS}²",
+    "135": UnitOfApparentPower.VOLT_AMPERE,  # Volt-Amp
+    "136": POWER_VOLT_AMPERE_REACTIVE,  # VAR = Volt-Amp Reactive
+    "137": "",  # NTP DateTime - Number of seconds since 1900
```

Comment:
How is empty string handled? We should never actually set an empty string as unit of measurement in the entity state.